frontend/express/public/javascripts/countly/vue/components/vis.js

```diff
@@ -1593,6 +1593,138 @@
                     </div>'
     }));
 
+    Vue.component("cly-map-picker", countlyVue.components.create({
+        components: {
+            'l-map': Vue2Leaflet.LMap,
+            'l-tile-layer': Vue2Leaflet.LTileLayer,
+            'l-marker': Vue2Leaflet.LMarker,
+            'l-circle': Vue2Leaflet.LCircle,
+        },
+        props: {
+            value: {
+                type: [Object],
+                required: false,
+                validator: function(item) {
+                    return item && item.lng && item.lat;
+                },
```

Comment:
This validator might cause errors (plus, there is this `required: false` and `default: null`). For async binding occassions, there will be time spans in which the value will be falsy making validator fn fail.

Reply:
I am not sure I follow in terms of async binding. When required is set to false, validator with false values passes.